python-manual/modules/ROOT/pages/install.adoc

```diff
@@ -0,0 +1,47 @@
+= Installation
+
+To start creating a Neo4j Python application, you first need to install the Python Driver and get a Neo4j database instance to connect to.
+
+[#install-driver]
+== Install the driver
+
+Use `pip` to install the link:https://pypi.org/project/neo4j/[Neo4j Python Driver] (requires Python >= 3.7):
+
+[source,bash]
+----
+pip install neo4j
+----
+
+Always use the latest version of the driver, as it will always work both with the previous Neo4j <<LTS>> release and with the current and next major releases.
+The latest `5.x` driver supports connection to any Neo4j 5 and 4.4 instance, and will also be compatible with Neo4j 6. For a detailed list of changes across versions, see the link:https://github.com/neo4j/neo4j-python-driver/wiki/5.x-changelog[driver's changelog].
```

Comment:
```suggestion
The latest `5.x` driver supports connection to any Neo4j 5 and 4.4 instances, and is also compatible with Neo4j 6. For a detailed list of changes across versions, see the link:https://github.com/neo4j/neo4j-python-driver/wiki/5.x-changelog[driver's changelog].
```
Neo4j 6? It's not even on the horizon yet.

Reply:
Forward and backward compatibility is important for developers, to have the guarantee that whatever work they invest today will be valid as long as possible. The Drivers team makes this commitment by guaranteeing that if you invest in developing with the driver v5, you can keep your code as-is for not only v5, but also for the next major. The same has happened with driver v4: it works with Neo 5.